sqldelight-gradle-plugin/src/main/kotlin/app/cash/sqldelight/gradle/VerifyMigrationTask.kt

```diff
@@ -113,10 +120,11 @@ abstract class VerifyMigrationTask : SqlDelightWorkerTask() {
     }
 
     override fun execute() {
+      ServiceLoader.load(DriverInitializer::class.java).firstOrNull()?.execute(parameters.properties.get())
```

Comment:
No objectives against the current implementation, but what about passing the connection properties here?

Otherwise, we have to add a test to not break passing the connection properties using `DriverManager.getConnection` somehow.

Reply:
sounds good, made the change!